src/app/models/medicalList.model.ts

```diff
@@ -0,0 +1,9 @@
+export class MedicalList {
```

Comment:
This model should be named as Treatment or MedicalRecord

Reply:
Alright, I will rename it now.